lib/src/native/wrapper/wrapper.h

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+#ifndef WRAPPER_H_
+#define WRAPPER_H_
+
+#include "../include/dart/dart_api_dl.h"
+#include "../include/cronet/cronet.idl_c.h"
+
+
+#include"wrapper_export.h"
+
+#include<stdbool.h>
+
+#ifdef __linux__
+  #include <dlfcn.h>
+  #define LIBTYPE void*
+  #define OPENLIB(libname) dlopen((libname), RTLD_NOW)

```

Comment:
This is only needed for your C++ testing code, and not for using the wrapper with `dart:ffi`, consider putting it in an `#ifdef`.